pkg/filter/proxy/request.go

```diff
@@ -80,6 +82,17 @@ func (p *pool) newRequest(
 		stdr.Host = r.Host()
 	}
 
+	// golang http.NewRequestWithContext will set ContentLength when body is bytes.Buffer,
```

Comment:
Cool, but if you want to comment on its behavior, we had better use the official reference.

Reply:
Agreed, I'm also curious about what's said in the Golang office document.

Reply:
updated!